README.rst

```diff
@@ -13,37 +13,58 @@ This benchmark is dedicated to solver of the ResNet classification fitting probl
 where :math:`i` is the sample index, :math:`x_i` is the input image, :math:`y_i` is the sample label, and :math:`L` is the cross-entropy loss function.
 
 
-Install
+Use
 --------
 
 This benchmark can be run using the following commands:
 
 .. code-block::
 
    $ pip install -U benchopt
-   $ git clone https://github.com/zaccharieramzi/benchmark_resnet_classif
+   $ git clone https://github.com/benchopt/benchmark_resnet_classif
    $ benchopt run benchmark_resnet_classif
 
-Apart from the problem, options can be passed to `benchopt run`, to restrict the benchmarks to some solvers or datasets, e.g.:
+While this command would run the entire benchmark, which includes several models, datasets and solvers, sequentially,
+you can restrict the run to a specific model, dataset and/or solver by passing the corresponding arguments.
+For example if I want to run the benchmark for the ResNet18 model on CIFAR10 dataset with the Adam solver, without a validation set, and for a single random seed:
 
 .. code-block::
 
-	$ benchopt run benchmark_resnet_classif -s solver1 -d dataset2 --max-runs 10 --n-repetitions 10
+	$ benchopt run benchmark_resnet_classif -o "*18" -d "cifar[*random_state=42*with_validation=False]" -s "adam-torch[batch_size=128,coupled_weight_decay=0.0,data_aug=True,decoupled_weight_decay=0.02,*,lr_schedule=cosine]"  --max-runs 200 --n-repetitions 1
```

Comment:
With the newest version of `benchopt`, I think this will not work. you need to run:
```suggestion
	$ benchopt run benchmark_resnet_classif -o "*[model_type=resnet,model_size=18]" -d "cifar[random_state=42,with_validation=False]" -s "adam-torch[batch_size=128,coupled_weight_decay=0.0,data_aug=True,decoupled_weight_decay=0.02,lr_schedule=cosine]"  --max-runs 200 --n-repetitions 1
```

Reply:
and not simply `-o "[model_size=18]"` ?

Reply:
No I think this would not work. You need the `*` to match the name.
For only using `model_size=18`, I am not sure what this would give actually? Have you tried?